internal/provider/schema_cloudflare_access_application.go

```diff
@@ -121,6 +123,28 @@ func resourceCloudflareAccessApplicationSchema() map[string]*schema.Schema {
 				},
 			},
 		},
+		"saas_app": {
```

Comment:
all these fields need to be documented with `Description` fields to generate the documentation

Reply:
Done